lib/smoke_detector/providers/rollbar.rb

```diff
@@ -44,14 +44,16 @@ def default_client_settings
         </script>
       JS
+      @client_monitoring_code = @client_monitoring_code.freeze
```

Comment:
`.freeze` should be called on the original string def (ending on line 55) so it will pull the string from the frozen string table rather than allocating a new one on every request.